### ISSUE TITLE
Update Scale_Redist Version to 1.2

### ISSUE THREAD

**Issue by Lisias** [net-lisias-ksp/TweakScale#276]
Reasoning:

Some internal interfaces should be made public, otherwise 3rd Parties would never have full support from TweakScale for some borderline situations.

**Comment by Lisias** [comment]
Implemented by https://github.com/net-lisias-ksp/TweakScale/commit/9c8e631b4bf42e09140a13e400fec955b8ad2730

**Comment by Lisias** [comment]
The last implementation wasn't enough.

Code was reworked on commit https://github.com/net-lisias-ksp/TweakScale/commit/6efd45081acd7d7219644a15686f5f249c613194 .

**Comment by Lisias** [comment]
The last implementation wasn't enough.

Code was reworked on commit https://github.com/net-lisias-ksp/TweakScale/commit/36d703508b055d7698a853dbd5227a22ab27273c

**Comment by Lisias** [comment]
V2.5 was too much. Bumping the Major Number would imply that people would need to recompile the World against the new DLL, what's wrong.

There're **new** things on Scale_Redist, but the ones that were always there didn't changed a bit - so there's no need to recompile anything, and so bumping the Major is unnecessary.

Bumping it to 1.2 instead, since I had already bumped it to 1.1 when I implemented a Interface for 3rd Parties Sanity Checks.
